src/main/java/Iris/ToDoList.java

```diff
@@ -0,0 +1,157 @@
+package iris;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks in the Iris application.
+ */
+public class ToDoList {
```

Comment:
Perhaps a better class name is TaskList because it does not only contain ToDo but also Event and Deadline. You can look into the Refactor feature in IntelliJ to do this easily.